README.md

```diff
@@ -83,6 +83,7 @@ The `containerTemplate` is a template of container that will be added to the pod
 * **command** The command the container will execute.
 * **args** The arugments passed to the command.
 * **ttyEnabled** Flag to mark that tty should be enabled.
+* **ports** Expose ports on the container.
```

Comment:
@derfred this works great, the only thing that you may want to add to the readme is to outline that you access a port that is open on another container you need to use `127.0.0.1` for the url, as doing it name based is not working for me.

Reply:
Found the solution, described on the PR